### PR TITLE
Authorizable comment action for proposals

### PR DESCRIPTION
**Pull request by verarojman**
https://github.com/decidim/decidim/pull/6916

### CODE REVIEW

**Comment by microstudi** [comment]
@verarojman  I think these are the only changes we need, let's discuss tomorrow:
https://github.com/Platoniq/decidim/pull/36/commits/dcbeacd0dc7e9996a1ca028c0c6ed223cb0b10a2

**Comment by verarojman** [comment]
@microstudi how do you think we should handle voting comments? The approach is not the same as just commenting. In my  [last commit](https://github.com/Platoniq/decidim/pull/36/commits/9f3704386b665b61e71baa05d63d08b619ab9b32) I tried to use the `action_authorized_button_to` but ran into this error. Any ideas? 🙏🏽 

image

File: decidim-comments/app/cells/decidim/comments/comment/votes.erb [hunk 1]
```diff
@@ -1,11 +1,11 @@
 <div class="comment__votes">
   <% if user_signed_in? %>
-    <%= button_to decidim_comments.comment_votes_path(model, weight: 1), remote: true, disabled: voted_down?, class: votes_up_classes, title: t("decidim.components.up_vote_button.text") do %>
+    <%= action_authorized_button_to decidim_comments.comment_votes_path(model, weight: 1), resource: model.commentable, remote: true, disabled: voted_down?, class: votes_up_classes, title: t("decidim.components.up_vote_button.text") do %>
```
Comment (microstudi) [inline decidim-comments/app/cells/decidim/comments/comment/votes.erb]: what about this:
we should put this in a helper in the cell.
If the commentable has a component, add the current_component context, if not just use the normal `button_to`

**Comment by microstudi** [comment]
no need for this to stay here anymore, :point_right: decidim#6916

**Pull request closed without merging**